### DISCUSSION TITLE
Discord Problem

### DISCUSSION THREAD

**Discussion by kphs2** [w-okada/voice-changer#712] [General]
Hello there i want to ask why when i use this in discord everytime my friend speak it will change his voice into a female voice and the sound would come out of my mic and my friends will hear their voice in a female voice. Can anyone tell me how to fix that.

**Comment by FernThatCodes** [comment]
Could you provide more details about the setting of your discord audio and voice changer audio settings?

**Reply by kphs2** [reply]
![Capture](https://github.com/w-okada/voice-changer/assets/102755347/f0284b71-1ec2-4793-94be-f64348b8cd19)
![image](https://github.com/w-okada/voice-changer/assets/102755347/7f636be9-225d-4f08-99de-48aa4da01c21)
![image](https://github.com/w-okada/voice-changer/assets/102755347/de1a388c-cda4-4010-a00e-634de212a930)
![image](https://github.com/w-okada/voice-changer/assets/102755347/387aaa51-c07a-405a-a2a9-ecdb66ac78ac)
![image](https://github.com/w-okada/voice-changer/assets/102755347/0a747ec1-4e10-4b77-b8b8-f4fbfd670b9e)
![image](https://github.com/w-okada/voice-changer/assets/102755347/76c325b8-c6cd-4a3a-844f-cf7b92e9aa3c)
*All my recording devices is 2 channel 24bit , 48000hz studio quality
*I tried to change my audio settings many times and it still didnt fix it
*I also tried to use Voice Meter to change settings and also doesnt work

**Comment by FernThatCodes** [comment]
Try changing the input in discord to the virtual audio cable and output device a speaker or a headphone, hope this helps!

**Reply by kphs2** [reply]
I tried your method and it still doesnt work :( When my friend speak his voice still become a female voice and he can hear his voice through my mic on discord. It would be great if there is any other way to fix this and thank you for your respond

**Reply by FernThatCodes** [reply]
sorry it didn't help....welp...I have no idea now
